CONTRIBUTING.md

```diff
@@ -0,0 +1,6 @@
+## How to contribute to guides.rubygems.org
```

Comment:
Maybe we can put this in `.github` so we don’t have to rename the existing file?

Reply:
I thought about that, and I know you suggested it on the other thread, but I feel like putting it at the root is more of the "standard" so it's more likely to be discovered this way, and if it's more likely to be discovered people are more likely to follow it.

Also on https://github.com/rubygems/guides/community - I believe that page would forever link "Contributing" to the old file, which is not desired behavior.